Updated readme as mentioned in #38

rsomlette opened the pull request:
## Description

Updating the readme as mentioned in https://github.com/kushthedude/To-Do-App/issues/38

kushthedude commented:
Please correct the typo in the PR

kushthedude commented:
Also I have written incorrect command to start FE Server, can you change it to `npm start`.
Thanks

rsomlette commented:
@kushthedude I fixed the typo and changed the command to install & start. 
(As you have a file yarn.lock, it makes more sense to use yarn instead of npm.)
